spec/services/exporter_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+describe Exporter do
+  subject { described_class.new }
+
+  EXPECTED_FILE_METADATA_KEYS = %i[id name content_type metadata].freeze
```

Comment:
why freeze if you already added a comment on line 1

Reply:
rubocop asked to freeze it